docs/api-docs/getting-started/api-status-codes.md

```diff
@@ -30,15 +30,16 @@ The BigCommerce API responds to requests with different HTTP status codes depend
 
 | Code | Text | Purpose |
 |-|-|-|
-| **400** | **Bad Request** | Issued when a malformed request was sent.
+| **400** | **Bad Request** | Issued when a malformed request was sent. |

```

Comment:
```suggestion
| **400** | **Bad Request** | Issued when a malformed request was sent. <br /> The request can not be completed due to a URL restriction. Check the URL for ports that may conflict with site permissions. |
```

Reply:
Added suggested commit to stay in line with other instances of table structure in other articles.